apache/superset
# chore(Dashboard): Disable save button in Native Filters when an error is present

### SUMMARY
This PR disabled the save button when an error is present in the native filters as it was suggested in this comment https://github.com/apache/superset/pull/16940#pullrequestreview-775022121

### AFTER
https://user-images.githubusercontent.com/60598000/136585167-40b9e4fd-41db-4b13-af1c-5d6cdfefde76.mp4

### TESTING INSTRUCTIONS
1. Open a Dashboard
2. Produce an error in the native filters form
3. Make sure the save button is disabled

### ADDITIONAL INFORMATION
